debian/control

```diff
@@ -4,53 +4,24 @@ Section: misc
 Priority: optional
 Standards-Version: 3.9.2
 Build-Depends:
- clang-3.8,
```

Comment:
Is this not necessary for adbshell.c?

https://github.com/google/android-cuttlefish/blob/master/host/commands/adbshell/adbshell.c

Though maybe gcc should be listed as a build dependency rather than clang, the Makefile looks like it is implicitly using gcc.

Reply:
Yeah, looks like I missed this because the buster build script is installing build-essential so gcc is being used.  I'll add gcc here.